helm-charts/common/llm-uservice/values.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for llm-uservice.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+hfToken: "insert-your-huggingface-token-here"
```

Comment:
please align with the environment variable name in README.md , like " HUGGINGFACEHUB_API_TOKEN"